source/fixed-length-array.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+Create a type that represents an array of provided types and of a provided fixed length. The arrays length and Array prototype mutations that manipulate its length are excluded in the resulting type.
+
```

Comment:
> Write about some real-world use-cases where it can be useful. (It can be hard sometimes for users to see where they would use something) - https://github.com/sindresorhus/type-fest/blob/master/.github/contributing.md

---

> If there has been any discussion somewhere about this type, include a link to it. For example, a discussion on the TypeScript issue tracker.